website/static/service-worker.js

```diff
@@ -3,7 +3,7 @@
 
 "use strict";
 
-importScripts("lib/sw-toolbox.js");
+importScripts("https://unpkg.com/sw-toolbox@3.6.0/sw-toolbox.js");
```

Comment:
This might not work — the same-origin policy also applies to `importScripts`.

Reply:
I tested on Chrome, macOS

Reply:
Does it work here? https://deploy-preview-4453--prettier.netlify.com/

Reply:
Yes of course

![image](https://user-images.githubusercontent.com/1574588/39829914-49bc5e54-5396-11e8-81e9-7eafa3474dbd.png)

Reply:
Worst thing that can happen is it'll stop working offline

Reply:
Seems legit! :D